nkms_eth/config.py

```diff
@@ -6,13 +7,58 @@
 import nkms_eth
 
 
+class NuCypherTokenConfig:
+    __subdigits = 18
+    _M = 10 ** __subdigits
+    __premine = int(1e9) * _M
+    __saturation = int(1e10) * _M
+    _reward = __saturation - __premine
```

Comment:
Where do we have the [maximum] mining rate? Feels like a good place for that

Reply:
Perhaps I'm a little underscore happy here... :-) Looks like it's in `NuCypherMinerConfig`